### PR TITLE
Show a message over the analytics panel if no data is available

### PR DESCRIPTION
**Pull request by spryor**
# Type of change? 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
Currently, if the analytics are empty, either nothing or the previous data will be shown. This change will hide the old data.

### CODE REVIEW

File: src/components/pages/dashboard/panels/analytics/analyticsPanel.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ export class AnalyticsPanel extends Component {
   handleWindowFocus = () => this.setState({ renderChart: true });
 
   componentDidMount() {
-    this.barChart = new this.tsiClient.ux.BarChart(document.getElementById(barChartId));
+    this.barChart = new this.tsiClient.ux.BarChart(document.getElementById(barChartId+'wrong'));
```
Comment (avanikp) [inline src/components/pages/dashboard/panels/analytics/analyticsPanel.js]: nit: Add space before and after +

File: src/components/pages/dashboard/panels/analytics/analyticsPanel.js [hunk 2]
```diff
@@ -121,6 +122,10 @@ export class AnalyticsPanel extends Component {
               }
             </div>
           </div>
+          {
+            (!showOverlay && !topAlerts.length && !Object.keys(alertsPerDeviceId).length)
```
Comment (avanikp) [inline src/components/pages/dashboard/panels/analytics/analyticsPanel.js]: can topAlerts be undefined ? Need a check for that ?
Same for Object.keys(alertsPerDeviceId)

File: src/components/pages/dashboard/panels/analytics/analyticsPanel.js (hunk 1 above)
Comment (ppathan) [reply src/components/pages/dashboard/panels/analytics/analyticsPanel.js]: Why is this `wrong` hardcoded?

File: src/components/pages/dashboard/panels/analytics/analyticsPanel.js (hunk 2 above)
Comment (spryor) [reply src/components/pages/dashboard/panels/analytics/analyticsPanel.js]: No, they are always defined in the dashboard.

**Pull request merged**